src/pages/home/ReportScreen.js

```diff
@@ -237,71 +234,76 @@ class ReportScreen extends React.Component {
                 style={[styles.appContent, styles.flex1, {marginTop: this.state.viewportOffsetTop}]}
                 keyboardAvoidingViewBehavior={Platform.OS === 'android' ? '' : 'padding'}
             >
-                <OfflineWithFeedback
-                    pendingAction={addWorkspaceRoomPendingAction}
-                    errors={addWorkspaceRoomErrors}
-                    errorRowStyles={styles.dNone}
+                <FullPageNotFoundView
+                    shouldShow={!this.props.report.reportID}
```

Comment:
When we receive the event from the server to set the report to `null` (when you lose access to the chat), if you were looking at the chat, then that's when we hit this flow. At first coded this to check if report was `null` and it was not working, when I debugged it, I saw this is what we receive right after the event is sent:
<img width="283" alt="image" src="https://user-images.githubusercontent.com/521248/193285952-0600b0fe-4909-470a-8834-75468083a9c8.png">
Not quite sure why those properties are there when that happens though

Reply:
@iwiznia I am quite sure this is [causing a deploy blocker](https://github.com/Expensify/App/issues/11503). For some reason, the reportID is not accessible and then the `shouldShow` is true for a split of a second when creating a report

Reply:
Late reply, but I think it could be better use something more reliable like `report.isAccessible: false` vs. a `report` not having a `reportID`. 

Perhaps there are other unexpected reasons why the report has no `reportID` and then instead of being intentional about when to show this "not found" view we will show this view randomly.

Reply:
>Not quite sure why those properties are there when that happens though

Yeah, I think I would feel better about the change we made in this PR if we did understand why those properties are there and leave some kind of comment so that the intention is clear.